forge-gui/res/cardsfolder/upcoming/saheeli_rai.txt

```diff
@@ -0,0 +1,9 @@
+Name:Saheeli Rai
```

Comment:
This is a card from Kaladesh right ? https://scryfall.com/card/kld/186/saheeli-rai

Reply:
no clue

Reply:
Yep, already in Forge

Reply:
upcoming is my default folder for my IDE

Reply:
If you updated Saheeli Rai, it would need to be moved from upcoming to its respective folder in "s"